.env.example

```diff
@@ -50,8 +50,8 @@ LLM_BACKEND=watsonx
 # Embedding backend, possible values are: ollama, openai, watsonx, bam, caikit
 EMBEDDING_BACKEND=watsonx
 
-# Extraction backend, possible values are: unstructured-opensource, unstructured-api
-EXTRACTION_BACKEND=unstructured-opensource
+# Extraction backend, possible values are: docling, unstructured-opensource, unstructured-api
+EXTRACTION_BACKEND=docling
```

Comment:
Please change all `:` to `=` in the `.env.example`. I got some weird issue when running `start:dev:workers:python`:

Why is it trying to parse `.env`?

```
> bee-api@0.0.12 start:dev:workers:python /Users/radek/Code/bee-api
> bash -c 'cd workers/python && poetry run python python/main.py'

Python-dotenv could not parse statement starting at line 93
Traceback (most recent call last):
  File "/Users/radek/Code/bee-api/workers/python/python/main.py", line 84, in <module>
    asyncio.run(main())
  File "/Users/radek/.pyenv/versions/3.11.7/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/Users/radek/.pyenv/versions/3.11.7/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/.pyenv/versions/3.11.7/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/radek/Code/bee-api/workers/python/python/main.py", line 75, in main
    workers, stop_workers = await create_workers()
                            ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/Code/bee-api/workers/python/python/main.py", line 39, in create_workers
    runners = await run_workers(config.run_bullmq_workers)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/Code/bee-api/workers/python/python/workers.py", line 56, in run_workers
    import extraction.extraction
  File "/Users/radek/Code/bee-api/workers/python/python/extraction/extraction.py", line 19, in <module>
    from database import database
  File "/Users/radek/Code/bee-api/workers/python/python/database.py", line 27, in <module>
    client = AsyncMongoClient(
             ^^^^^^^^^^^^^^^^^
  File "/Users/radek/Library/Caches/pypoetry/virtualenvs/non-package-mode-3L7xjKIj-py3.11/lib/python3.11/site-packages/pymongo/asynchronous/mongo_client.py", line 843, in __init__
    self._options = options = ClientOptions(username, password, dbase, opts, _IS_SYNC)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/Library/Caches/pypoetry/virtualenvs/non-package-mode-3L7xjKIj-py3.11/lib/python3.11/site-packages/pymongo/client_options.py", line 219, in __init__
    self.__pool_options = _parse_pool_options(username, password, database, options, is_sync)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/Library/Caches/pypoetry/virtualenvs/non-package-mode-3L7xjKIj-py3.11/lib/python3.11/site-packages/pymongo/client_options.py", line 170, in _parse_pool_options
    ssl_context, tls_allow_invalid_hostnames = _parse_ssl_options(options)
                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/radek/Library/Caches/pypoetry/virtualenvs/non-package-mode-3L7xjKIj-py3.11/lib/python3.11/site-packages/pymongo/client_options.py", line 133, in _parse_ssl_options
    ctx = get_ssl_context(
          ^^^^^^^^^^^^^^^^
  File "/Users/radek/Library/Caches/pypoetry/virtualenvs/non-package-mode-3L7xjKIj-py3.11/lib/python3.11/site-packages/pymongo/ssl_support.py", line 101, in get_ssl_context
    ctx.load_verify_locations(ca_certs)
ssl.SSLError: [X509: NO_CERTIFICATE_OR_CRL_FOUND] no certificate or crl found (_ssl.c:4154)
```

Reply:
definitely 👍

Reply:
Also comment out this line, it's detecting a value [here](https://github.com/i-am-bee/bee-api/blob/3ebd1b1a7a2a562f6969ae97eb015655ece2056d/workers/python/python/database.py#L22):

```
MONGODB_CA_CERT= # optional
```